locales-src/newtab.ftl

```diff
@@ -185,7 +185,7 @@ newtab-discovery-empty-section-topstories-timed-out = Oops! We almost loaded thi
 # This is shown at the bottom of the trending stories section and precedes a list of links to popular topics.
 newtab-pocket-read-more = Popular Topics:
 newtab-pocket-more-recommendations = More Recommendations
-newtab-pocket-how-it-works = How it works
+newtab-pocket-how-it-works = What's Pocket?
```

Comment:
String changes require new IDs
https://firefox-source-docs.mozilla.org/intl/l10n/l10n/fluent_review.html#changes-to-existing-messages

Also, `’` instead of `'`